client_src/js/printerManager/printer-constructor.js

```diff
@@ -0,0 +1,323 @@
+import Validate from "../lib/functions/validate";
+import UI from "../lib/functions/ui";
+import OctoFarmClient from "../lib/octofarm_client.js";
+
+let newPrintersIndex = 0;
+
+const removeLine = function (element) {
+  element.remove();
+};
+class Printer {
+  constructor(printerURL, camURL, apikey, group, name) {
+    this.settingsAppearance = {
+      color: "default",
+      colorTransparent: false,
+      defaultLanguage: "_default",
+      name,
+      showFahrenheitAlso: false
+    };
+    this.printerURL = printerURL;
+    this.camURL = camURL;
+    this.apikey = apikey;
+    this.group = group;
+  }
+}
+export class PrintersManagement {
+  constructor(printerURL, camURL, apikey, group, name) {
```

Comment:
I see a constructor for a single printer's properties in a manager pattern... pls dont

Reply:
This was mainly aimed at splitting the god awful printerManagement file down not at refactoring function as mentioned. Outside of scope of PR, please keep within scope.